### PR TITLE
[ID] People

### PR DESCRIPTION
**Pull request by fajar13k**
- Add Bahasa Indonesia translation of `People` article
- Update Bahasa Indonesia translation of `Community Contributors` article
- Add Bahasa Indonesia translation of `Users with unique titles` article

### CODE REVIEW

**Comment by vzrenggamani** [comment]
Due to size of the PR is quite big, and the contents is very _dynamic_ I can't review everything OwO, but I can say its okay to merge.

File: wiki/People/Users_with_unique_titles/id.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+Pemenang [MWC 4K 2019](/wiki/Tournaments/MWC/2019_4K) dengan gelar pengguna **osu!mania Champion**:
+
+- ![][flag_KR] [blueBloody](https://osu.ppy.sh/users/8046839)
+- ![][flag_KR] [Kalkai](https://osu.ppy.sh/users/10790649)
+- ![][flag_KR] [SuddenDeath](https://osu.ppy.sh/users/6699923)
+- ![][flag_KR] [Transcendence](https://osu.ppy.sh/users/903155)
+- ![][flag_KR] [wonder5193](https://osu.ppy.sh/users/8474029)
+- ![][flag_KR] [Yeoul](https://osu.ppy.sh/users/4698728)
+
+### Pemenang kontes pemetaan umum
```
Comment (troke12) [inline wiki/People/Users_with_unique_titles/id.md:48]: Pemenang kontes General Mapping, kayaknya aneh kalau pakai pemetaan umum (?)

File: wiki/People/Users_with_unique_titles/id.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+- ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057)
+- ![][flag_LY] [Soul Evans](https://osu.ppy.sh/users/4490770)
+- ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340)
+- ![][flag_KR] [W h i t e](https://osu.ppy.sh/users/685229)
+- ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661)
+- ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768)
+
+### Nominator Elit
+
+Sejak 2020, dalam mendukung sistem pemeringkatan terhadap aktivitas anggota [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) dan [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), mereka telah dianugerahi dengan gelar pengguna **Elite Nominator**. Gelar ini diberikan pada awal setiap tahun terhadap kontribusi yang dibuat dan berlaku sekali saja ketika berada dalam tim, dan akan dihapus pada saat pengguna tersebut meninggalkan tim.
```
Comment (troke12) [inline wiki/People/Users_with_unique_titles/id.md:83]: kayaknya `pengguna` di >dengan gelar pengguna **Elite ... gk perlu diisi

File: wiki/People/Users_with_unique_titles/id.md (hunk 1 above)
Comment (fajar13k) [reply wiki/People/Users_with_unique_titles/id.md:48]: Ohya, kayanya kebawa terjemahan ini. 🆗

File: wiki/People/Users_with_unique_titles/id.md (hunk 2 above)
Comment (fajar13k) [reply wiki/People/Users_with_unique_titles/id.md:83]: Sip 👍

File: wiki/People/Users_with_unique_titles/id.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+
+### Pemenang kontes osu!remix
+
+Kontes osu!remix menghadiahkan gelar **osu!mixer** sebagai hadiah utama dari kontes tersebut.
+
+- ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471)
+
+### Artis unggulan
+
+[Featured Artists](/wiki/Featured_Artists) with osu! accounts are given the **Featured Artist** title to show their involvement.
```
Comment (gagahpangeran) [inline wiki/People/Users_with_unique_titles/id.md:112]: Baris ini lupa dihapus?

**Pull request merged**